src/main/java/com/amplitude/Amplitude.java

```diff
@@ -206,4 +207,16 @@ private void scheduleFlushEvents() {
       flushThread.start();
     }
   }
+
+  public void debug(String message) {
+    logger.log(logger.getTimeStamp(), message);
```

Comment:
Why not just `log(message)`? What is the point of exposing the time as a parameter?

Reply:
I thought timestamp can help user further analyze what happened during certain period of time, like average logged/flushed events/second. I found that the logger.error was used like logger.error("Invalid API Key", exception.getMessage()) . Should I keep the same format here without timestamp?  Two parameters for log method may not be necessary.

Reply:
When you log a message, I would assume the timestamp is always just the current time, right? I would say passing a customized message is more useful.

Reply:
Agree, also I think what Qingzhuo mentioned make sense, we don't need to expose these log interface to customers.